docs/best-practices/accessibility.md

```diff
@@ -127,19 +127,20 @@ Users can navigate an application through headings. Having descriptive headings
 | form            | role="form"          | Collection of form-associated elements                                                                           |
 | section         | role="region"        | Content that is relevant and that users will likely want to navigate to. Label must be provided for this element |
 
-:::tip Tip:
+<box type="tip">
```

Comment:
You may have missed adding back the Tip header for this tip box!

Reply:
Oh thank you! Fixed